### PR TITLE
Add post-build usage info for Lua port.

### PR DESCRIPTION
**Pull request by paul-reilly**
Currently there are no instructions for using the Lua package after installation. This adds the necessary info.

- What does your PR fix? n/a

- Which triplets are supported/not supported? Have you updated the CI baseline?
n/a?

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
I believe so.

### CODE REVIEW

File: ports/lua/portfile.cmake [hunk 1]
```diff
@@ -51,4 +51,6 @@ vcpkg_copy_tool_dependencies(${CURRENT_PACKAGES_DIR}/tools/lua)
 
 # Handle copyright
 file(COPY ${CMAKE_CURRENT_LIST_DIR}/COPYRIGHT DESTINATION ${CURRENT_PACKAGES_DIR}/share/lua/copyright)
+# Copy post-build CMake instructions
+file(COPY ${CMAKE_CURRENT_LIST_DIR}/usage ${CURRENT_PACKAGES_DIR}/share/lua)
```
Comment (JackBoosY) [inline ports/lua/portfile.cmake]: Should be:
```
file(INSTALL ${CMAKE_CURRENT_LIST_DIR}/usage DESTINATION ${CURRENT_PACKAGES_DIR}/share/lua)
```

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: ports/lua/portfile.cmake (hunk 1 above)
Comment (paul-reilly) [reply ports/lua/portfile.cmake]: Omg thanks, that was it.

**Review by JackBoosY — CHANGES_REQUESTED** [review]
Can you update version info?
See [documentation](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md#update-the-version-field-in-the-control-file-of-any-modified-ports).

**Comment by paul-reilly** [comment]
Of course sorry, that's it bumped up.

**Review by JackBoosY — APPROVED** [review]

**Comment by vicroms** [comment]
Thanks for the PR!

**Pull request merged**